packages/color-area/stories/color-area.stories.ts

```diff
@@ -58,6 +59,16 @@ export const Default = ({ onChange, onInput }: StoryArgs): TemplateResult => {
     `;
 };
 
+export const appliedValues = (): TemplateResult => {
+    return html`
+        <sp-color-area
+            .color=${{ space: 'hsv', coords: [250, 90, 80] }}
```

Comment:
Property binding is used in the first <sp-color-area> to pass a complex object as the color property. This ensures that the component receives the color value in a structured format that it can use internally. I can remove it as well but then instead of `.color=${{ space: 'hsv', coords: [250, 90, 80] }}` I would have to use `color= hsv(250,90,80)` which you can see in the subsequent lines. This story is to demonstrate that we can pass color as a css string, as a colorjs object and also as deconstructed values